crates/nu-command/src/commands/pathvar/append.rs

```diff
@@ -0,0 +1,55 @@
+use crate::prelude::*;
+use nu_engine::WholeStreamCommand;
+use nu_errors::ShellError;
+use nu_protocol::{Signature, SyntaxShape};
+use nu_source::Tagged;
+use nu_test_support::NATIVE_PATH_ENV_VAR;
+use std::path::PathBuf;
+
+pub struct SubCommand;
+
+impl WholeStreamCommand for SubCommand {
+    fn name(&self) -> &str {
+        "pathvar append"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("pathvar append").required("path", SyntaxShape::FilePath, "path to append")
+    }
+
+    fn usage(&self) -> &str {
+        "Add a path to the end of the pathvar"
+    }
+
+    fn run(&self, args: CommandArgs) -> Result<OutputStream, ShellError> {
+        add(args)
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Append /bin to the pathvar",
+            example: "pathvar append /bin",
+            result: None,
+        }]
+    }
+}
+
+pub fn add(args: CommandArgs) -> Result<OutputStream, ShellError> {
+    let ctx = &args.context;
+    let path_to_append_arg: Tagged<PathBuf> = args.req(0)?;
+    let path_to_append = path_to_append_arg.item.into_os_string().into_string();
+
+    if let Ok(path) = path_to_append {
+        let mut pathvar = ctx.scope.get_env(NATIVE_PATH_ENV_VAR).unwrap();
+        pathvar.push(':');
```

Comment:
Won't this break Windows functionality?

You may need something like this, that exists in the `char` command. Or prehaps relocate it to somewhere more common that everyone can use.

```rust
// Character used to separate directories in a Path Environment variable on windows is ";"
#[cfg(target_family = "windows")]
const ENV_PATH_SEPARATOR_CHAR: char = ';';
// Character used to separate directories in a Path Environment variable on linux/mac/unix is ":"
#[cfg(not(target_family = "windows"))]
const ENV_PATH_SEPARATOR_CHAR: char = ':';
```

Reply:
Oh, I didn't know windows had a different separator. I'll fix that. Thanks.